src/Microsoft.AspNetCore.Blazor.Cli/Server/Startup.cs

```diff
@@ -69,7 +69,7 @@ private static string FindClientAssemblyPath(IApplicationBuilder app)
         {
             var env = app.ApplicationServices.GetRequiredService<IHostingEnvironment>();
             var contentRoot = env.ContentRootPath;
-            var binDir = FindClientBinDir(contentRoot);
+            var binDir = AppContext.BaseDirectory;
```

Comment:
I don't think this is right. The `BaseDirectory` would refer to the CLI tool, not the application you're launching. Or if you've verified this correctly picks out the correct `binDir` for the app being launched (in both debug and release modes) please let me know!

Reply:
Good point. This happens to work in our tests because we're using Microsoft.AspNetCore.Blazor.Cli as a ProjectReference. For those using this tool as  DotNetCliToolReference, BaseDirectory points to the wrong place.

To fix this properly, you'll need to [implement a technique](https://natemcmaster.com/blog/2016/12/26/project-evalutation-cli-tool/) we use in dotnet-watch and others to scrape project information.

I'll revert this and apply a different workaround to get tests passing.